src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;

```

Comment:
The `SupplyOperation` class should be placed in the `strategy` package to follow the project structure guidelines.